net/unbound/files/unbound.init

```diff
@@ -6,6 +6,9 @@ START=61
 USE_PROCD=1
 
 start_service() {
+	chmod -R u=rwX,g=rX,o= /etc/unbound
+	chown -R unbound:unbound /etc/unbound
+
```

Comment:
My implementation from [about 24 hours ago](https://github.com/openwrt/packages/pull/2834/commits/414eaacd90c655aa6ae8ac5c0b74ece435b7061c) does not use postinst at all (beside the built-in handling of `USERID`). Did you look at it?